### PR TITLE
[NEP-3777] Ability to lazily get collections from Select and Multi-Select filters

### PR DESCRIPTION
**Pull request by msxavi**
https://jobready.atlassian.net/browse/NEP-3777

lazy_filters

### CODE REVIEW

**Comment by andrba** [comment]
lazy components duplicate lots of methods of existing components. They also duplicate code between each other. What can be done to keep the code DRY?

File: src/components/FilterBar/FilterDisplay/Inputs/LazyMultiSelectInput.react.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    filter.value = event.target.value;
+  }
+
+  render() {
+    return (
+      <li>
+        <input
+          className="form-control"
+          value={this.state.value}
+          ref="reactLazyMultiSelect"
```
Comment (andrba) [inline src/components/FilterBar/FilterDisplay/Inputs/LazyMultiSelectInput.react.js:83]: should it have a `multiple` attribute?
how does this component populate itself in a case of a saved search?

**Comment by msxavi** [comment]
@andrba My JS and React skills are poor, therefore, I'm not entitled to tell you what can be done to dry it a bit more. Do u have any tips?

File: src/components/FilterBar/FilterDisplay/Inputs/LazyMultiSelectInput.react.js (hunk 1 above)
Comment (msxavi) [reply src/components/FilterBar/FilterDisplay/Inputs/LazyMultiSelectInput.react.js:83]: It's currently working without it. See below. Regarding saved search, I'm not sure how it is   done either, I suppose it must be tested within Neptune, right?

lazy_filters

**Pull request merged**